crates/frontend/src/pages/config_version.rs

```diff
@@ -0,0 +1,77 @@
+use crate::api::fetch_config;
+use crate::components::alert::{Alert, AlertType};
+use crate::components::skeleton::Skeleton;
+use crate::components::toast::Toast;
+use leptos::*;
+use leptos_router::use_params_map;
+
+#[component]
+pub fn config_version() -> impl IntoView {
+    let params = use_params_map();
+    let tenant = params.with(|p| p.get("tenant").cloned().unwrap_or_default());
+    let version = params.with(|p| p.get("version").cloned());
+
+    let config_resource = create_blocking_resource(
+        move || (tenant.clone(), version.clone()),
+        |(tenant, version)| async move { fetch_config(tenant, version).await },
+    );
+
+    view! {
+        <div class="p-8">
+
+            {
+                view! {
```

Comment:
Can you remove this? The code was moved out, but then this nesting of block and then a view is not required.